t/ui/18-tests-details.t

```diff
@@ -57,7 +58,8 @@ t::ui::PhantomTest::wait_for_ajax;
 ok($driver->find_element('#preview_container_out', 'css')->is_displayed(), "preview window opens on click");
 like($driver->find_element('#preview_container_in', 'css')->get_text(), qr/wait_serial expected/, "Preview text with wait_serial output shown");
 $driver->find_element('[title="wait_serial"]', 'css')->click();
-t::ui::PhantomTest::wait_for_ajax;
+# wait for animation to finish
+sleep .2;
```

Comment:
Of course you can be lucky without wait_for_ajax - if the ajax call is quicker than the selenium side of it. But it's actually the right thing to do - click on a link that triggers ajax and wait for the ajax action to be finished before testing your expectations.

With animations it's different - we're not waiting for the server side, we're waiting for the animation to be finished before we can verify the element is hidden. And that animation is 150 ms, so waiting 200 sounded like a reasonable approach. In that respect, sleeping 0.1 will most likely cause random failures and sleeping 1000 seconds will work but sounds a bit too pessimisitic.